src/ui/public/doc_table/components/table_row.js

```diff
@@ -77,6 +78,35 @@ module.directive('kbnTableRow', function ($compile) {
 
         $detailsScope.row = $scope.row;
 
+        $detailsScope.copyTextToClipboard = text => {
```

Comment:
Thanks. I think I can use that to write my own module service. Can you provide an example of `debounce` being used in another file? It doesn't appear to be imported like other modules. (remember I'm an angular noob).